server/src/internalClusterTest/java/org/elasticsearch/snapshots/SnapshotStatusApisIT.java

```diff
@@ -261,8 +262,9 @@ public void testCorrectCountsForDoneShards() throws Exception {
             indexTwo
         );
         assertThat(snapshotShardStateAfterNodeRestart.getStage(), is(SnapshotIndexShardStage.DONE));
-        assertThat(snapshotShardStateAfterNodeRestart.getStats().getTotalFileCount(), equalTo(totalFiles));
-        assertThat(snapshotShardStateAfterNodeRestart.getStats().getTotalSize(), equalTo(totalFileSize));
+        assertNotNull("expected a description string for missing stats", snapshotShardStateAfterNodeRestart.getDescription());
```

Comment:
Could you check that the other shard snapshot does not have a description field?